obip-ts-reports.md

```diff
@@ -0,0 +1,71 @@
+__OBIP__: TS-Reports
```

Comment:
Header formatting from OBIP guidance (OBIP1) doesn't have each line starting in bold and has the complete header within HTML <pre> </pre> tags

Reply:
I like this better, but you're right. I'll change it and perhaps propose a change and see how people like it.